### DISCUSSION TITLE
No matching camera found/OpenCV.Net.Native.NativeMethods error when I try to run Pylon Capture

### DISCUSSION THREAD

**Discussion by SaiK238** [bonsai-rx/bonsai#1600] [Q&A]
Is anyone familiar with this error? 
I tried connecting the Basler Camera (BASLER acA1920 - 155 um and BASLER acA1440 - 220 um) via USB link to the Pylon Capture node and it isn't working. Once I load the PFS file, the serial number of the Basler Camera is not visible in the drop-down menu and when I try to run the code, I get the "No matching camera found" error. If I put the serial number in the tab manually, I get a type initializer error for OpenCV.Net.Native.NativeMethods. 
I am using Pylon Viewer version 7.1 and Bonsai- Pylon Capture version 0.3.0.  
I have already deleted and clean downloaded Bonsai, have the Visual C++ 2010, 2012, 2013 and 2015-22 redistributable installed and also tried downgrading OpenCV.Net to 3.3.1 from 3.4.2 but I'm still getting the same error. 
I also tried reducing the frame rate and image resolution, but nothing is working. 
![Capture2](https://github.com/bonsai-rx/bonsai/assets/135891727/1552c826-5cd3-4a21-85d7-f3bfcfe42efe)
![error](https://github.com/bonsai-rx/bonsai/assets/135891727/940ad512-8b23-403d-ba4a-5849fd223a2c)

**Comment by chiyu1203** [comment]
Do you keep you pylon viewer closed when you run the bonsai file?
